src/static/css/2019.css

```diff
@@ -586,6 +587,7 @@ header nav {
 
 header nav > ul > li > * {
   margin-right: 50px;
+  margin-right: 2vw;
```

Comment:
What's this?

Reply:
It gives a slightly smaller or larger margin for larger viewports without having to introduce a new breakpoint.

Between 900px and 1200px it really needs to be 20px to avoid the two like ToC happening very early (<1120px) but that's a little squashed when we have more space on desktop so used `vw` units instead so there's a little more room to breath when we have the space.

I left the old 50px as a fallback but probably can remove that as `vw` [support looks to be pretty comprehensive](https://caniuse.com/#search=vw)